Fix the old makefile (WPA_Supplicant-0.6.4) and transfer just the configvars.
But I have no idea what they are for...

diff -ruN wpa_supplicant-0.6.9/wpa_supplicant/Makefile wpa_supplicant-0.6.9.new/wpa_supplicant/Makefile
--- wpa_supplicant-0.6.9/wpa_supplicant/Makefile	2009-03-23 15:06:28.000000000 +0100
+++ wpa_supplicant-0.6.9.new/wpa_supplicant/Makefile	2009-11-13 03:52:51.000000000 +0100
@@ -36,6 +37,11 @@
 	echo CONFIG_DRIVER_HOSTAP=y >> .config
 	echo CONFIG_DRIVER_WEXT=y >> .config
 	echo CONFIG_WIRELESS_EXTENSION=y >> .config
+	echo CONFIG_DRIVER_NDISWRAPPER=y >> .config
+	echo CONFIG_EAP_PSK=y >> .config
+	echo CONFIG_EAP_TLS=y >> .config
+	echo CONFIG_EAP_MD5=y >> .config
+	echo CONFIG_CTRL_IFACE=y >> .config
 
 install: all
 	mkdir -p $(DESTDIR)$(BINDIR)
